llvm/test/Transforms/GVNSink/sink-ignore-dbg-intrinsics.ll

```diff
@@ -25,9 +27,9 @@ if.else:                                          ; preds = %entry
   br label %if.end
 
 ; CHECK-LABEL: if.end:
-; CHECK: [[SINK:%.*.sink]] = phi i32 [ %a, %if.then ], [ %b, %if.else ]
-; CHECK: %add = add nsw i32 [[SINK]], 1
-; CHECK: %xor = xor i32 %add, 1
+; CHECK: [[SINK:%.*sink]] = phi i32 [ {{.*}} ], [ {{.*}} ]
```

Comment:
nit: we should have a fixed order here.
maybe swap the PHI operands and see if that passed the check.

Reply:
or better use the `llvm-project/llvm/utils/update_test_checks.py` to update the test

Reply:
Thanks! Now the incoming blocks has a fixed order.